src/state/opamSpdxList.ml

```diff
@@ -436,4 +436,5 @@ let exceptions = OpamStd.String.Set.of_list @@ List.map String.lowercase_ascii @
   "GCC-exception-2.0";
   "mif-exception";
   "OCaml-LGPL-linking-exception";
+  "LGPL-3.0-linking-exception";
```

Comment:
This file is autogenerated so this isn't quite the right fix (there is a comment in the dune file)
However https://github.com/ocaml/opam/pull/4768 should fix that for good hopefully.